screens/AddCryptoScreen/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, useState } from 'react';
+import { Text, View, SafeAreaView, TouchableOpacity, TextInput } from 'react-native';
+import { useDispatch } from 'react-redux';
+import styles from './styles';
+import { addCripto } from '../../store/actions/crypto.actions';
+import colors from '../../assets/colors/colors';
+import { Routes } from '../../navigation/Routes';
+
+interface Props {
+    navigation: any,
+}
+
+const AddCryptoScreen: FC<Props> = ({ navigation }) => {
+
+    const dispatch = useDispatch()
+    const [textInput, setTextInput] = useState('');
+    const [focus, setFocus] = useState(false)
+
+    const handlerAddCoin = () => {
+        dispatch(addCripto(textInput))
+        navigation.navigate(Routes.HOME)
+    }
+
+    return (
+        <SafeAreaView>
+            <View style={styles.addCryptoContainer}>
+                <TouchableOpacity
+                    onPress={navigation.goBack.bind()}>
```

Comment:
[minor] In order to bind this function, use the correct syntax `navigation.goBack`. Should work as well.